engine/table/src/main/java/io/deephaven/engine/table/impl/MultiJoinStateManager.java

```diff
@@ -5,21 +5,21 @@
 import io.deephaven.engine.table.impl.util.WritableRowRedirection;
 
 /**
- * This is a common interface for the static and incremental state manager so that our bucketed MultiJoinTableImpl system is
+ * This is a common interface for the static and incremental state manager so that our bucketed MultiJoinTable system is
  * capable of using them interchangeably to build the table.
  */
 public interface MultiJoinStateManager {
     /**
-     * Add the given table to this multijoin result.
+     * Add the given table to this multiJoin result.
      *
      * @param table the table to add
      * @param sources the column sources that contain the keys
-     * @param tableNumber the table number that we are adding rows for
+     * @param tableNumber the table number for which we are adding rows
      */
     void build(final Table table, ColumnSource<?>[] sources, int tableNumber);
 
     /**
-     * How many rows are in our result table?
+     * Get the number of rows in the result table
```

Comment:
Is this overall or hash table size?

Reply:
This returns used hash table entry count (correct for static but incorrect for incremental).